xbmc/input/InputManager.cpp

```diff
@@ -394,9 +395,55 @@ bool CInputManager::OnEvent(XBMC_Event& newEvent)
   case XBMC_MOUSEBUTTONDOWN:
   case XBMC_MOUSEBUTTONUP:
   case XBMC_MOUSEMOTION:
-    m_Mouse.HandleEvent(newEvent);
-    ProcessMouse(g_windowManager.GetActiveWindowID());
+  {
+    bool handled = false;
+
+    if (newEvent.type == XBMC_MOUSEMOTION)
+    {
+      static int m_mouseStateX;
+      static int m_mouseStateY;
+      for (std::vector<MOUSE::IMouseHandler*>::iterator it = m_mouseHandlers.begin(); it != m_mouseHandlers.end(); ++it)
+      {
+        if ((*it)->OnMouseMotion(newEvent.motion.x - m_mouseStateX, newEvent.motion.y - m_mouseStateY))
```

Comment:
I couldn't find anthing that stores relative coordinates. there's xrel/yrel, but that's a different thing. From what I can see x/y are the absolute coordinate inside kodi window, xrel/yrel are the absolute coordinates relative to the screen 0,0 position. If kodi is running in fullscreen mode then they are the same. There are no other values set in https://github.com/xbmc/xbmc/blob/master/xbmc/windowing/WinEventsX11.cpp#L546. I could only find the relative position being calculated in MouseStat: https://github.com/xbmc/xbmc/blob/master/xbmc/input/MouseStat.cpp#L52.